addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -358,28 +358,21 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
      */
     _getContentEditableAreas() {
         const $savableZones = $(this.websiteService.pageDocument).find(this.savableSelector);
-        const $editableSavableZones = $savableZones
+        let $editableSavableZones = $savableZones
             .not('input, [data-oe-readonly], ' +
                  '[data-oe-type="monetary"], [data-oe-many2one-id], [data-oe-field="arch"]:empty')
             .filter((_, el) => {
                 return !$(el).closest('.o_not_editable').length;
             });
 
-        // TODO review in master. This stable fix restores the possibility to
-        // edit the company team snippet images on subsequent editions. Indeed
-        // this badly relies on the contenteditable="true" attribute being on
-        // those images but it is rightfully lost after the first save.
-        // grep: COMPANY_TEAM_CONTENTEDITABLE
-        let $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable img');
-
         // To make sure the selection remains bounded to the active tab,
         // each tab is made non editable while keeping its nested
         // oe_structure editable. This avoids having a selection range span
         // over all further inactive tabs when using Chrome.
         // grep: .s_tabs
-        $extraEditableZones = $extraEditableZones.add($editableSavableZones.find('.tab-pane > .oe_structure'));
+        $editableSavableZones = $editableSavableZones.add($editableSavableZones.find('.tab-pane > .oe_structure'));
```

Comment:
You could let it be a `const` and chain the `.add` call after the `.filter` call. It is possible to put the comment in between like this:
```js
const x = base
    .chain(...)
    .chain(...)
    // This is a special chain
    .chain(...);
```